HW0/P5/P5.txt

```diff
@@ -1 +1,33 @@
 # Your answers here.
+1. 255 s
+   2^7 + 2^6 + ... + 2^0
+
+2. Yes, faster. 34 s
```

Comment:
I don't agree with this. I see what you're trying to do, but at the end of the day, a single cashier can't paralyze this computation. Maybe if you explained in a little more detail what you have in mind?